frontend/src/app/modules/common/autocomplete/create-autocompleter.component.ts

```diff
@@ -35,53 +35,10 @@ import {PathHelperService} from "core-app/modules/common/path-helper/path-helper
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/db51ff98125ea1ed608d04d8bdfc3d3810baafed/frontend/src/app/modules/common/autocomplete/create-autocompleter.component.ts#L36"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>